api/controllers/MerchStoreController.ts

```diff
@@ -188,4 +197,16 @@ export class MerchStoreController {
     await this.merchStoreService.updateOrderItems(fulfillOrderRequest.items);
     return { error: null };
   }
+
+  private verifyMerchOrderRequest(orderRequest:MerchItemOptionAndQuantity[],
+    user:UserModel):MerchItemOptionAndQuantity[] {
+    if (!PermissionsService.canAccessMerchStore(user)) throw new ForbiddenError();
+    const originalOrder = orderRequest.filter((oi) => oi.quantity > 0);
+    const orderIsEmpty = originalOrder.reduce((x, n) => x + n.quantity, 0) === 0;
+    if (orderIsEmpty) throw new UserError('There are no items in this order');
+    const numUniqueUuids = (new Set(originalOrder.map((oi) => oi.option))).size;
+    if (originalOrder.length !== numUniqueUuids) throw new BadRequestError('There are duplicate items in this order');
```

Comment:
IMO this is more of a UserError than a BadRequestError. I think BadRequestError should be for invalid requests, and this is something that'd be a UserError since it's a valid request. @shravanhariharan2 yay/nay?
```suggestion
    if (originalOrder.length !== numUniqueUuids) throw new UserError('There are duplicate items in this order');
```